Newsletters: add 323 (2024-10-04) (#1929)
bitcoinops/bitcoinops.github.io: _posts/en/newsletters/2024-10-04-newsletter.md

---
title: 'Bitcoin Optech Newsletter #323'
permalink: /en/newsletters/2024/10/04/
name: 2024-10-04-newsletter
slug: 2024-10-04-newsletter
type: newsletter
layout: newsletter
lang: en
---
This week's newsletter FIXME

## News

*No significant news this week was found in any of our [sources][optech
sources].*

## Releases and release candidates

*New releases and release candidates for popular Bitcoin infrastructure
projects.  Please consider upgrading to new releases or helping to test
release candidates.*

> - [BDK 1.0.0-beta.4][] is a release candidate of this library for

Reviewer: [1.0.0-beta.5](https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.5) was published in the meantime.

  funding or [splice][topic splicing] is created using the fee credits. Clients
  rely on liquidity providers to honour the fee credits in future transactions.

- [LDK #3303][] adds a new `PaymentId` for inbound payments to improve
  idempotent event handling. This allows users to easily check if an event has
  already been handled when replaying events during node restarts. The risk of
  duplicate processing that was possible when relying on the `PaymentHash` is
  eliminated. The `PaymentId` is a hash-based message authentication code (HMAC)
  of the [HTLC][topic htlc] channel identifier and HTLC identifier pairs
  included in the payment.

- [BDK #1616][] signals opt-in [RBF][topic rbf] by default in the
  `TxBuilder` class.  The caller can disable the signal by changing the
  sequence number.

- [BIPs #1600][] makes several changes to the [BIP85][] specification, including
  specifying that `drng_reader.read` (responsible for reading random numbers) is
  a first-class function rather than an evaluation. This update also clarifies
  endianness handling, adds support for [testnet][topic testnet], includes a new

>   reference implementation in Python, clarifies that a [HD wallet][topic bip32]

Reviewer 2: ```suggestion
  reference implementation in Python, clarifies that an [HD wallet][topic bip32]
```

>   seed wallet import format (WIF) uses most significant bits, adds the

Reviewer 2: ```suggestion
  seed wallet import format (WIF) uses the most significant bits, adds the
```

---
title: 'Bitcoin Optech Newsletter #323'
permalink: /en/newsletters/2024/10/04/
name: 2024-10-04-newsletter
slug: 2024-10-04-newsletter
type: newsletter
layout: newsletter
lang: en
---
This week's newsletter FIXME

## News

*No significant news this week was found in any of our [sources][optech
sources].*

## Releases and release candidates

*New releases and release candidates for popular Bitcoin infrastructure
projects.  Please consider upgrading to new releases or helping to test
release candidates.*

- [BDK 1.0.0-beta.4][] is a release candidate of this library for

>   building wallets and other Bitcoin-enabled applications.  The original

Reviewer 3: rephrasing idea, feel free to ignore

```suggestion
  building wallets and other Bitcoin-enabled applications.  Changes in this release candidate include renaming the `bdk` Rust crate to `bdk_wallet` and extracting lower-layer modules to their own crates, like `bdk_chain`, ...
```
